docs/knowledge-base/DevOps/pipelines/release-management.mdx

```diff
@@ -0,0 +1,112 @@
+---
+title: Release Management
+sidebar_position: 3
+toc_min_heading_level: 2
+toc_max_heading_level: 6
+
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Release Management Guide
+
+This document outlines the release management process for Libot Mipuy, which involves multiple teams and services. 
+The release strategy follows semantic versioning ([SemVer](https://semver.org/)) for individual services
+and "Libot version" for managing deployed Libot stack on networks.
+
+## Libot Version Tag
+
+- The **Libot Version** is assigned based on the `helm-charts` release version.
+- This version represents the bundle of all service releases for the deployment to production.
+
+## Service Branching Strategy
+
+Each service must maintain at least one main branch to track the latest stable release and major version updates:
+
+1. **`master`**:
+   - Represents the latest stable version of the service.
+   - Merges to this branch indicate the most recent production-ready release.
+   
+2. **`MAJOR.x.x`** (e.g., `1.x.x`, `2.x.x`):
+   - Tracks the major version series of a service.
+   - A new branch is created for every major version that is smaller than the latest major version.
+   - Example:
+     - If a service's latest release version is `3.3.0` and a new bug fix needs to be merged to latest release with major version '2', 
+       a pull request (PR) will be created for both:
+       - `master` (represents 3.x.x) - only if the bug fix relevant to this release too
+       - `2.x.x` (to track the new version with the fix).
+     - Once merged, both branches will represent a release with the bug fix, while the `1.x.x` branch remains unchanged.
+
+## Service Release Flow
+
+![Releasing Flow](./img/release-management/release-on-repo.png)
+
+Explanation:
+
+1. The developer creates a PR
+
+2. On merging to master: 
+    - release-please takes action and opens a PR release with these changes:
+        - Updating package.json to the new release version
+        - Updating Changelog with the new version
+        - Updating helm (chart version)
+
+    :::info
+    The release version will be based on ([SemVer](https://semver.org/)) (according to the PR title)
+    :::
+
+3. On merging the release PR to master:
+    - release-please will create a new release
+
+4. On new release, an automate workflow will be triggered to:
```

Comment:
Please mention that a `PR` is opened to update the `helm charts` with the new version (I guess using release please?).
I know it's also mentioned at the next step, but at lease reference it somehow.